<?xml version="1.0"?>
<launch>
	<node pkg="vicon_pose" type="vicon_pose" name="conv_q_p_Controller">
		<param name="object_name" value="wand" type="str" />
		<remap from= "subject_pose" to="controller_pose" />
	</node>
	
	<node pkg="vicon_pose" type="vicon_pose" name="conv_q_p_subject">
		<param name="object_name" value="GREEN_BEE" type="str" />
	</node>

	<node pkg="vicon_pose" type="vicon_pose" name="conv_q_p_UAV">
		<param name="object_name" value="GREEN_BEE" type="str" />
		<remap from= "subject_pose" to="UAV_pose" />
	</node>

	<node pkg="demo_UAV_controller" type="demo_UAV_controller" name="demo_UAV_controller">
	</node>

	<include  file="$(find demo_UAV_controller)/launch/mitAscTec_vicon.launch" />
</launch>
